daemon/cmd/daemon_main.go

```diff
@@ -533,6 +534,15 @@ func init() {
 	flags.String(option.NodePortMode, option.NodePortModeSNAT, "BPF NodePort mode (\"snat\", \"dsr\", \"hybrid\")")
 	option.BindEnv(option.NodePortMode)
 
+	flags.String(option.NodePortAlg, option.NodePortAlgRandom, "BPF load balancing algorithm (\"random\", \"maglev\")")
+	option.BindEnv(option.NodePortAlg)
+
+	flags.Uint(option.MaglevTableSize, maglev.DefaultTableSize, "Maglev per service backend table size (parameter M)")
+	option.BindEnv(option.MaglevTableSize)
+
+	flags.String(option.MaglevHashSeed, maglev.DefaultHashSeed, "Maglev cluster-wide hash seed (base64 encoded)")
```

Comment:
nit: If these settings live in `pkg/default` like the other defaults, then `daemon_main.go` doesn't need to also import `maglev`.

Reply:
My personal preference is to have defaults in a package they belong to for better readability. However, if a consensus to list all defaults in `pkg/defaults` then I can move the default there.